src/__init__.py

```diff
@@ -139,15 +141,18 @@ def run_migrate(clients: Clients, user_inputs: UserInputs) -> dict[str, str]:
             user_inputs.commit_sha,
             DOCUMENTATION_TAG,
         )
+        # If a PR is open we should probably close the PR
```

Comment:
Perhaps `Given there are diffs compared to the base, if a PR is open, it should be closed`?

Reply:
I'd say

```Given there are NO diffs compared to the base, if a PR is open, it should be closed```